### PR TITLE
Adds overload to TransactionDSL.command to allow signers to be passed as List, not array.

### PR DESCRIPTION
**Pull request by joeldudleyr3**

### CODE REVIEW

File: test-utils/src/main/kotlin/net/corda/testing/TransactionDSLInterpreter.kt [hunk 1]
```diff
@@ -111,6 +111,13 @@ class TransactionDSL<out T : TransactionDSLInterpreter>(val interpreter: T) : Tr
 
     /**
      * @see TransactionDSLInterpreter._command
+     * Allows signers to be passed in as a List rather than an Array from Java.
```
Comment (shamsasari) [inline test-utils/src/main/kotlin/net/corda/testing/TransactionDSLInterpreter.kt]: This is not restricted to Java, Kotlin will benefit from this as well

File: test-utils/src/main/kotlin/net/corda/testing/TransactionDSLInterpreter.kt [hunk 2]
```diff
@@ -111,6 +111,13 @@ class TransactionDSL<out T : TransactionDSLInterpreter>(val interpreter: T) : Tr
...
+     */
+    fun command(signers: List<PublicKey>, commandDataClosure: () -> CommandData) =
+            _command(signers, commandDataClosure())
```
Comment (shamsasari) [inline test-utils/src/main/kotlin/net/corda/testing/TransactionDSLInterpreter.kt]: Why do we have this underscore method? Perhaps this should be deprecated as it does the same thing as this new one.

**Comment by joeldudleyr3** [comment]
Back to you, Shams.

**Comment by joeldudleyr3** [comment]
@shamsasari Mike said the init cycle was still a real possibility here. Not worth the improvements in usability IMO.

**Pull request closed without merging**